src/_posts/platform/getting-started/2000-01-01-getting-started-with-modsecurity.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Getting started with ModSecurity on Scalingo
```

Comment:
nitpick:

```suggestion
title: Getting Started With ModSecurity on Scalingo
```

In English, usually titles have the first letter of all words upper cased. I usually use https://capitalizemytitle.com/ with the Wiki style.